### PR TITLE
General additions

### PR DESCRIPTION
**Pull request by gall9488**
Expanded the nec_tables.py entries and updated dictionary references in the osd.py file.

### CODE REVIEW

File: nec_tables.py [hunk 1]
```diff
@@ -460,178 +460,434 @@
...
+                         '350': 0.039,
+                         '400': 0.035,
+                         '500': 0.029,
+                         '600': 0.025,
+                         '750': 0.021,
+                         '1000': 0.018}}
+
+# Aluminum conductor resistance (R) per 1000 FEET from NEC 2017 Chapter 9, Table 9
+# XX:{YY: ZZ}  :: XX=conduit_material, YY=cond_size, ZZ=cond_reactance
+cond_resistance_al =  {'PVC':{
```
Comment (bt-) [inline nec_tables.py]: Should the aluminum abbreviation be consistent? The abbreviation used for the key in the conductor resistance dictionary is 'Alum'.  Should this be `cond_resistance_alum`?

File: nec_tables.py [hunk 2]
```diff
@@ -438,4 +438,200 @@
                  5000: {'Cu': '700', 'Al': '1200'},
                  6000: {'Cu': '800', 'Al': '1200'}}
 
-conduit_type = []
+# Percent of Cross Section of conduit for Conductors and Cables from NEC 2017 Chapter 9, Table 1
+# (XX,YY)  :: XX=conductor/cable count, YY=conduit_xsection max fill
+condit_xsection =  [(1,0.53),
```
Comment (bt-) [inline nec_tables.py]: Typo here `condit` instead of `conduit`

File: nec_tables.py [hunk 3]
```diff
@@ -438,4 +438,200 @@
...
+conduit_type = ['EMT',
+                'ENT',
+                'FMC',
+                'IMC',
+                'LFNC-A',
+                'LFNC-B',
+                'LFNC-C',
+                'LFMC',
+                'RMC',
+                'PVC80'.
```
Comment (bt-) [inline nec_tables.py]: Typo here `.` (period) instead of `,` (comma)

File: nec_tables.py [hunk 4]
```diff
@@ -438,4 +438,200 @@
...
+                        1.00:0.959,
+                        1.25:1.647,
+                        1.50:2.225,
+                        2.00:3.630,
+                        2.50:5.135,
+                        3.00:7.922,
+                        3.50:10.584,
+                        4.00:13.631},
+            # Article 356.2(1) - Liquidtight Flexible Nonmetallic Conduit (LFNC-A)
+                 'LFNC-A':{
```
Comment (bt-) [inline nec_tables.py]: change dash to underscore

File: nec_tables.py [hunk 5]
```diff
@@ -438,4 +438,200 @@
...
+                        1.50:1.986,
+                        2.00:3.291,
+                        2.50:4.695,
+                        3.00:7.268,
+                        3.50:9.737,
+                        4.00:12.554,
+                        5.00:19.761,
+                        6.00:28.567},
+            # Article 352 - Rigid PVC Conduit, Type A
+                 'PVC A':{
```
Comment (bt-) [inline nec_tables.py]: Change space to dash `PVC-A`

File: nec_tables.py [hunk 6]
```diff
@@ -438,4 +438,200 @@
...
+                        1.00:1.085,
+                        1.25:1.767,
+                        1.50:2.324,
+                        2.00:3.647,
+                        2.50:5.453,
+                        3.00:8.194,
+                        3.50:10.694,
+                        4.00:13.723},
+            # Article 352 - Rigid PVC Conduit, Type EB
+                 'PVC EB':{
```
Comment (bt-) [inline nec_tables.py]: Change space to dash `PVC-EB`

File: nec_tables.py [hunk 7]
```diff
@@ -438,4 +438,200 @@
...
+                        2.00:3.874,
+                        3.00:8.709,
+                        3.50:11.365,
+                        4.00:14.448,
+                        5.00:22.195,
+                        6.00:31.530}}
+
+# Conduit material for AC resistance values
+# (XX,YY)  :: XX=conduit_type, YY=conduit_material
+conduit_material = [('EMT','Aluminum'),
```
Comment (bt-) [inline nec_tables.py]: I think this might be better as a dictionary than a list of tuples.

File: nec_tables.py [hunk 8]
```diff
@@ -438,4 +438,456 @@
                  5000: {'Cu': '700', 'Al': '1200'},
                  6000: {'Cu': '800', 'Al': '1200'}}
 
-conduit_type = []
+# Percent of Cross Section of conduit for Conductors and Cables from NEC 2017 Chapter 9, Table 1
+# (XX,YY)  :: XX=conductor/cable count, YY=conduit_xsection max fill
```
Comment (bt-) [inline nec_tables.py]: I think you could simplify your 'legends' by just putting `(conductor/cable count, conduit_xsection maxfill)`

File: nec_tables.py (hunk 1 above)
Comment (gall9488) [reply nec_tables.py]: Changed to alum

File: nec_tables.py (hunk 2 above)
Comment (gall9488) [reply nec_tables.py]: Corrected.

File: nec_tables.py (hunk 3 above)
Comment (gall9488) [reply nec_tables.py]: Corrected.

File: nec_tables.py (hunk 5 above)
Comment (gall9488) [reply nec_tables.py]: Corrected.

File: nec_tables.py (hunk 6 above)
Comment (gall9488) [reply nec_tables.py]: Corrected.

File: nec_tables.py (hunk 7 above)
Comment (gall9488) [reply nec_tables.py]: Converted to dictionary.

File: nec_tables.py (hunk 8 above)
Comment (gall9488) [reply nec_tables.py]: Simplified all legends.

**Comment by gall9488** [comment]
Chapter 9, Table 8 and Chapter 9, Table 9 have been added to the nec_tables.py file. This todo can be reviewed and closed.

File: calc_flow_v000.ipynb [hunk 9]
```diff
@@ -0,0 +1,514 @@
...
+    "print(list_range)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in ocpd_check:\n",
```
Comment (bt-) [inline calc_flow_v000.ipynb:458]: With this syntax i is not the integer location of the list.  It is the value of the list.  If you wanted to get an integer python has the handy enumerate() function, but in this case you can just use the returned value and skip the indexing.

Change i to something like `ocpd_size`.

File: calc_flow_v000.ipynb [hunk 10]
```diff
@@ -0,0 +1,514 @@
...
+    "    if ocpd_check[i] >= (design_current * 1.25):\n",
```
Comment (bt-) [inline calc_flow_v000.ipynb:459]: `if ocpd_size >= (design_current * 1.25):`

File: calc_flow_v000.ipynb [hunk 11]
```diff
@@ -0,0 +1,514 @@
...
+    "        ocpd_rating = ocpd_check[i]\n",
```
Comment (bt-) [inline calc_flow_v000.ipynb:460]: `ocpd_rating = ocpd_size`

File: calc_flow_v000.ipynb (hunk 9 above)
Comment (gall9488) [reply calc_flow_v000.ipynb:458]: Corrected. It worked - thanks!

File: calc_flow_v000.ipynb (hunk 10 above)
Comment (gall9488) [reply calc_flow_v000.ipynb:459]: Corrected. It worked - thanks!

File: calc_flow_v000.ipynb (hunk 11 above)
Comment (gall9488) [reply calc_flow_v000.ipynb:460]: Corrected. It worked - thanks!

**Pull request merged**